chromium_src/components/search_engines/brave_template_url_service_util_unittest.cc

```diff
@@ -19,6 +20,12 @@
 #include "components/sync_preferences/testing_pref_service_syncable.h"
 #include "testing/gtest/include/gtest/gtest.h"
 
+namespace TemplateURLPrepopulateData {
+  extern void LocalizeEngineList(
```

Comment:
why not include the header?

Reply:
This was a weird one; it's defined in a CC file and I tried putting the extern inside the header. However, no targets use it (outside of the module) so it gets flagged as a non-use and creates error (since tests are different target)